### PR TITLE
Adds the Kodiak-Class Mobile Command Vehicle

### PR DESCRIPTION
**Pull request by AtlasRules**
## About The Pull Request
Adds the Kodiak-class Mobile Command Vehicle, for when you need to command and conquer the battlefield from a distance.

FEATURES:
- Many guns!
- Requistions!
- Too many Command & Conquer references!
- A TEG!
- Large quantities of swanky uniforms!
- The Tacitus! (note: just alien alloy, not some alien bible thing)
-  Research and Development! Now in cargo and security flavours!

Images: 
image
image (SDMM View)
And here's a pic of Titanium Alloy, the new material that is used LITERALLY only for these walls. 
image

Also adds funny tan walls and fixes the syndie maid outfit box haha hohee, also some more ACLF jobs

Thanks to Apogee, Prefab, Tetra, Sylveon and Typebar (who did the TEG) for mapping help and any% for the funny wall sprites

TO DO:
- [x] - Fix the walls sometimes just making the turf below it vanish
- [x] - Make the actual fucking map itself
- [x] - Fix falsewalls runtiming when you try to build them, and fix their open and close animations
- [x] - Probably some optimisation and other shit I've missed
- [x] - Add a sprite for titanium alloy

## Why It's Good For The Game
more ship, also first of the nu-walls 

## Changelog
:cl:
add: The Kodiak-class Mobile Command Vehicle, a new ship for the ACLF. 
add: Titanium alloy! It's literally only used for titanium alloy walls.
imageadd: tan walls and tialloy sprite, tanwalls by any%, tialloy sheet sprite by atlas
bugfix: syndie maid box now actually spawns with the uniforms
code: adds some ACLF jobs
/:cl:

### CODE REVIEW

**Comment by AtlasRules** [comment]
why the fuck was orc code in here i am so upset and angry

**Comment by MusselMain** [comment]
What even does this ship do? As it stands faction fleets aren't really a thing yet, and most ships fly independent anyways, so I'm a bit confused how this role will work.

**Comment by AtlasRules** [comment]
> What even does this ship do? As it stands faction fleets aren't really a thing yet, and most ships fly independent anyways, so I'm a bit confused how this role will work.

ok so like you know the Kodiak from command & conquer tiberian sun

it's literally that but for some non-specific faction that i've yet to decide on yet

**Comment by AtlasRules** [comment]
right now, it'll just function like the Carina, Colossus, Tartarus and other similar ships

**Comment by Latentish** [comment]
Please add a screenshot of your ship and update it as you work on the PR. Once "The actual Map is made" you'll be able to get better criticism. But I've been wondering what a "command" ship would look like here. So let's see what you do with it.

**Comment by AtlasRules** [comment]
the walls making shit vanish only happens when made on top of crates, which you can't do normally. happens to OTHER mineral walls too so i'll just make a bug report

**Comment by AtlasRules** [comment]
the falsewalls runtiming is apparently unrelated? it's forensics.dm fuckery and also happens in hyperspace too so uh yea
also i've fixed the icons for the falsewall

**Comment by AtlasRules** [comment]
this is (probably) playable now? if someone could remove WIP that'd be great ty

**Comment by Latentish** [comment]
While I'll need to look this over more carefully, at a glance this looks pretty solid.

**Comment by AtlasRules** [comment]
okay NOW it's fixed

**Comment by AtlasRules** [comment]
had to do some minor fiddling to get integ to be happy, so the pictures deviate slightly. all you need to know is there's no longer an input for the O2 or plasma and there's now SMESes in the nacelles

**Comment by Latentish** [comment]
I have the same problem with this ship as I had with the Hyena. Because as much as I do like this ship, it ultimately fits a similar role as the Hyena. It has that combat specialization, but instead of having a well equipped mining department, it has a well equipped engineering department (TEG is basically infinite power if you have limited plasma and oxy, or literally infinite power if you have atmos tiles for those gasses, which this ship does). The problem that the Hyena had was that it had cargo. It's just a bit too powerful. Maybe replace it with a deparmental techfab for cargo instead.
I also have a bit of an issue with the TEG + atmos tiles. I think another ship does this too, and I kind of have a problem with them as well. The downside of a TEG is that it's a fucking gas hog, so managing your gas output is a major part of using it. So I'd for sure like to see those atmos tiles you're using be replaced with atmos tanks will extra capacity (The rube uses 200k kpa tanks). Although, Apogee the other maptainer, may disagree with me.
Aside from those 2 things, this ship seems pretty solid! I like it quite a lot.

**Comment by AtlasRules** [comment]
can I get DNM and WIP removed? this is ready to be merged barring some minor changes

**Comment by triplezeta** [comment]
WIP removed, DNM remains for the time being however. I want to wait until the rest of any%'s walls are completed and merged before this gets merged. I've added the stale exempt label so that this stays open until that day comes.

**Review by triplezeta — APPROVED** [review]
speaking of walls, I approve of the walls in this image.

**Comment by AtlasRules** [comment]
also every ship i see with big atmo tanks (kugel, nemo, shetty) all use the floor/engine/[gas] floors, so uh, idk

**Comment by AtlasRules** [comment]
> WIP removed, DNM remains for the time being however. I want to wait until the rest of any%'s walls are completed and merged before this gets merged. I've added the stale exempt label so that this stays open until that day comes.

aight mk mk

**Comment by AtlasRules** [comment]
Hmm... I should kill MapDiffBot2...

**Comment by AtlasRules** [comment]
@triplezeta tm Pleage... 🥺

**Review by Latentish — APPROVED** [review]
Good ass ship.

**Comment by AtlasRules** [comment]
TGHHGASDHGIJHADGIH TFUYCJCING G PLASTITANIYUM VERSION BECAUSE PEOPLE ARE INFANTS FINE

**Comment by TheNeoGamer42** [comment]
So where are the C&C references aside the name, the vague shape that honestly resembles the shape of the version of the Kodiak In That One Game That Doesn't Exist, and the funny Tacitus, the totally not intentional naming of its type being MCV, and the walls that are sorta in the tib sun GDI beige? You say it has too many, and this is TOTALLY not enough man

**Comment by VaelophisNyx** [comment]
CABAL the secbot is another one

**Comment by AtlasRules** [comment]
> This pull request has conflicts, please resolve those before we can evaluate the pull request.

https://user-images.githubusercontent.com/49167264/175645420-a632da9d-6559-4c07-90a1-9d7879f7de5c.mp4

**Comment by AtlasRules** [comment]
ok SRM conflict fixed

**Comment by Latentish** [comment]
Are these walls merged yet?

**Comment by AtlasRules** [comment]
Right now it’s the plastitanium version, I’m waiting on wallening to happen before I switch back to the old ones

File: code/__DEFINES/icon_smoothing.dm
Comment (Fikou) [inline code/__DEFINES/icon_smoothing.dm:150]: This needs to follow the other defines and not break the smoothing system

File: code/game/objects/items/devices/PDA/PDA_types.dm
Comment (Fikou) [inline code/game/objects/items/devices/PDA/PDA_types.dm:164]: you are making pda/syndicate an empty path that does nothing, why?

File: code/game/objects/items/stacks/sheets/sheet_types.dm
Comment (Fikou) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:815]: format this like 
```
= list(
	thing1,
	thing2,
)
```

**Review by Fikou — CHANGES_REQUESTED** [review]
above

File: code/game/objects/items/devices/PDA/PDA_types.dm
Comment (AtlasRules) [reply code/game/objects/items/devices/PDA/PDA_types.dm:164]: it was a test to see if i could make a detatomix-less syndie pda but it didn't work and clearly, forgot to delete it

File: code/game/objects/items/stacks/sheets/sheet_types.dm
Comment (AtlasRules) [reply code/game/objects/items/stacks/sheets/sheet_types.dm:815]: no other composite material does it like this

File: code/__DEFINES/icon_smoothing.dm
Comment (AtlasRules) [reply code/__DEFINES/icon_smoothing.dm:150]: > This needs to follow the other defines and not break the smoothing system

it does though?

**Comment by Latentish** [comment]
What's the status on the pending change request, and the walls of this ship?

**Comment by AtlasRules** [comment]
don’t understand any of the requests and wallening has yet to occur

**Comment by ZephyrTFA** [comment]
Is this still being worked on? And why is this stale exempt

**Comment by AtlasRules** [comment]
> Is this still being worked on? And why is this stale exempt

it’s functionally finished and wallening has yet to happen

**Comment by AtlasRules** [comment]
i don't understand any of the requests fikou made so i'm just gonna ask for another review

**Comment by Latentish** [comment]
I will request that you add some clutter to the hallways and that engineering room. There's a lot of somewhat empty space that could use beautifying. Other than that this seems a bit like a gunboat with a lot of strengths in other departments as well. I'm unsure how the balancing of this ship looks at the moment, but perhaps try cutting back supplies for the various departments around the ship.

**Comment by AtlasRules** [comment]
okay so this literally cannot be added until nuwalls are done, can this please be re-opened and just kept under stale exempt? if not i get why but uh

yea

**Pull request closed without merging**
